### PR TITLE
Displayed Category of Scones

### PR DESCRIPTION
**Pull request by TOTBWF**
# Description

This PR adds the displayed category of scones, and also proves that change of base of a fibration yields a fibration.

## Notes

The motivation for the category of scones could be a lot better! Ideally we'd prove canonicity of STLC using it, but I have too many WIP branches rn, so l'll circle back to it at a later date :)

### CODE REVIEW

File: src/Cat/Displayed/Instances/Scone.lagda.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+Morphisms behave like they do in the slice category of $\ca{B}$:
+given a map $f : X \to Y$ in $\ca{B}$, a map over $f$ in the category
+between $(U, su : U \to \ca{B}(1, X))$ and $(V, sv : V \to \ca{B}(1, Y))$
+consists of a function $uv : U \to V$, such that for all $u : U$,
+we have $f \circ su(u) = sv (uv u)$.
+
+We can make the exposition out of the way, we can now define the
+category of scones. We do this by abstract nonsense, by performing
+a base change along the global sections functor of the slice category
+of $\ca{B}$.
```
Comment (plt-amy) [inline src/Cat/Displayed/Instances/Scone.lagda.md:46]: I think this should read something like:

> [Pull back] the [fundamental fibration] of Sets along the global sections functor of B, to a fibration over B

Aiui the situation is

where the two vertical arrows are fibrations?

Comment (plt-amy) [reply src/Cat/Displayed/Instances/Scone.lagda.md:46]:
```latex
\[\begin{tikzcd}
  {\mathrm{Scn}(\mathcal{B})} && {\mathrm{Arr}(\mathbf{Sets})} \\
  \\
  {\mathcal{B}} && {\mathbf{Sets}}
  \arrow["{\mathrm{Slices}(\mathbf{Sets})}", from=1-3, to=3-3]
  \arrow["{\mathcal{B}(*,-)}"', from=3-1, to=3-3]
  \arrow[lies over, dashed, from=1-1, to=3-1]
  \arrow[lies over, from=1-1, to=1-3]
  \arrow["\lrcorner"{anchor=center, pos=0.125}, draw=none, from=1-1, to=3-3]
\end{tikzcd}\]
```
if you want to include the diagram

File: src/Cat/Displayed/Instances/Scone.lagda.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+```agda
+scone : ∀ {X} → (U : Set ℓ) → (∣ U ∣ → Hom top X) → Scones.Ob[ X ]
+scone U s = cut {domain = U} s
+
+scone-hom : ∀ {X Y} {f : Hom X Y} {U V : Set ℓ}
+            → {su : ∣ U ∣ → Hom top X} {sv : ∣ V ∣ → Hom top Y}
+            → (uv : ∣ U ∣ → ∣ V ∣)
+            → (∀ u → f ∘ (su u) ≡ sv (uv u))
+            → Scones.Hom[ f ] (scone U su) (scone V sv)
```
Comment (plt-amy) [inline src/Cat/Displayed/Instances/Scone.lagda.md:72]: Align the arrows with the `:`, not the `∀`:
```suggestion
scone-hom : ∀ {X Y} {f : Hom X Y} {U V : Set ℓ}
          → {su : ∣ U ∣ → Hom top X} {sv : ∣ V ∣ → Hom top Y}
          → (uv : ∣ U ∣ → ∣ V ∣)
          → (∀ u → f ∘ (su u) ≡ sv (uv u))
          → Scones.Hom[ f ] (scone U su) (scone V sv)
```

File: src/Cat/Displayed/Instances/Scone.lagda.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+scone-hom uv p = slice-hom uv (funext p)
+```
+
+## As a fibration
+
+The category of scones over $\ca{B}$ is always a fibration. This is
+where our definition by abstract nonsense begins to shine: base change
+preserves fibrations, and the codomain fibration is a fibration whenever
+the base category has pullbacks, which sets has!
```
Comment (plt-amy) [inline src/Cat/Displayed/Instances/Scone.lagda.md:81]:
```suggestion
the base category has pullbacks, which $\sets$ [has]!

[has]: Cat.Instances.Sets.Complete.html#finite-set-limits
```

File: src/Cat/Displayed/Univalence/Thin.lagda.md [hunk 4]
```diff
@@ -186,3 +186,4 @@ Full-substructure _ R S embed Sst .∘-is-hom = Sst .∘-is-hom
 Full-substructure _ R S embed Sst .id-hom-unique α β =
   has-prop-fibres→injective (embed _ .fst) (embed _ .snd)
     (Sst .id-hom-unique α β)
+-->
```
Comment (plt-amy) [inline src/Cat/Displayed/Univalence/Thin.lagda.md:189]: lol thanks

**Review by plt-amy — CHANGES_REQUESTED** [review]

File: src/Cat/Displayed/Univalence/Thin.lagda.md (hunk 4 above)
Comment (TOTBWF) [reply src/Cat/Displayed/Univalence/Thin.lagda.md:190]: No worries haha

**Comment by TOTBWF** [comment]
Not totally sure why this is failing? I can't find the missing link it's referring to in `Proposition.Resizing`

**Pull request merged**